imagebuildah/build.go

```diff
@@ -1817,19 +1817,20 @@ func (s *StageExecutor) EnsureContainerPath(path string) error {
 	targetPath := filepath.Join(s.mountPoint, path)
 	fileDest, _ := os.Lstat(targetPath)
 	if fileDest != nil {
+		// If the file exists, check to see if it's a symlink.
+		// If it is a symlink, convert to it's target.
 		if fileDest.Mode()&os.ModeSymlink != 0 {
-			if symLink, err := resolveSymlink(s.mountPoint, path); err == nil {
-				targetPath = symLink
-			} else {
+			if targetPath, err := resolveSymlink(s.mountPoint, path); err != nil {
```

Comment:
Darn late night changes.  I did that because err complaining without the colon and didn't think about it changing the scope of targetPath.  I've declared err and removed the colon, but now that brings back the creation of /var/lib/tempest in the hosts directory.  I'm beginning to think I don't need to call resolveSymlink() here.  Need to play.

Reply:
OK, poking further, I over complicated this function with my previous checkin.  It looks like the only thing that needed to change was to change `os.Stat()` to `os.Lstate()` so that it could check a symbolic link if it was presented to it.